README.md

```diff
@@ -41,6 +41,11 @@ See our [Documentation](https://docs.meilisearch.com/guides/introduction/quick_s
 
 ## 🔧 Installation
 
+### Requirements <!-- omit in toc -->
+
+- Ruby version 2.6.0 or later
+- httparty >= 0.17.1
+
```

Comment:
I would rather add a simple sentence.

"This package requires Ruby version 2.6.0 or later"